src/RequestParser.php

```diff
@@ -99,26 +100,48 @@ protected function inlineFiles(Request $request): array
         /** @var string|null $mapParam */
         $mapParam = $request->post('map');
         if (null === $mapParam) {
-            throw new BadRequestGraphQLException('Could not find a valid map, be sure to conform to GraphQL multipart request specification: https://github.com/jaydenseric/graphql-multipart-request-spec');
+            throw new BadMultipartRequestGraphQLException('Could not find a valid map.');
```

Comment:
Add `@throws` in `parseRequest` - we should probably use https://phpstan.org/blog/bring-your-exceptions-under-control